src/pages/experimental/edit-pattern/index.mdx

```diff
@@ -0,0 +1,201 @@
+---
+title: Edit & Update Pattern
```

Comment:
If I was looking for Update pattern guidance, I think I'd probably click on Edit. I think we can safely call it the "Edit pattern" and call out Update in the introduction you already do. It'll still appear upon search if a user can't find it.